contributing-guides/maintainers-guide.md

```diff
@@ -58,20 +58,18 @@ as a guideline for current and future maintainers.
   (1) **pass the automated tests** (GitHub Actions, CLA signing, etc.),
   (2) have the **review comments addressed**, and
   (3) get **approved reviews by two maintainers**
-  (the second maintainer can perform the merge immediately after accepting.)
+  (4) are at least **24 hours** old unless changes are trivial
```

Comment:
Feel free to change the value, I've been using 24 hours but that is just me.

Reply:
I would suggest that both of these items are good to have here.